plan/plan.go

```diff
@@ -175,6 +176,23 @@ type AWSProvider struct {
 	RoleArn   *string     `yaml:"role_arn"`
 }
 
+func (a *AWSProvider) String() string {
+	accountId := a.AccountID.String()
```

Comment:
🚫 **[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>var `accountId` should be `accountID` (golint)